src/python/metrics/logs.py

```diff
@@ -252,14 +252,28 @@ def uncaught_exception_handler(exception_type, exception_value,
   sys.__excepthook__(exception_type, exception_value, exception_traceback)
 
 
+def configure_appengine():
+  """Configure logging for App Engine."""
+  logging.getLogger().setLevel(logging.INFO)
+
+  if os.getenv('LOCAL_DEVELOPMENT') or sys.version_info.major == 2:
```

Comment:
split into two checks to make it easy to not to remove os.getenv('LOCAL_DEVELOPMENT') check.

Reply:
more of a nit, can skip.